src/Native/Unix/System.Net.Http.Native/pal_easy.c

```diff
@@ -0,0 +1,360 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
btw, what is the rational of calling this file `pal_easy`? 360 lines of code in this file isn't that much easier when compared to other five liners c files. 😄

Reply:
btw, what is the rationale behind calling this file `pal_easy`? 360 lines of code in this file isn't that much easier when compared to other five liners c files. 😄